app/assets/stylesheets/petitions/_forms.scss

```diff
@@ -26,6 +26,10 @@ select.form-control:hover {
 .form-group {
   margin-bottom: $gutter;
   clear: both;
+
+  input[type=file] {
+    line-height: 1ex;
+  }
```

Comment:
I'm not sure how we usually do file uploads – I don't think we have a recommended pattern. If this works then it's fine.